packages/trie/package.json

```diff
@@ -29,6 +29,7 @@
     "lint": "../../config/cli/lint.sh",
     "lint:fix": "../../config/cli/lint-fix.sh",
     "tsc": "../../config/cli/ts-compile.sh",
+    "tape": "tape -r ts-node/register",
```

Comment:
Already asked this in #1399, where is this new `tape` command coming from and why is this necessary?